recipes/hklpy2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "hklpy2" %}
+{% set version = "0.0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hklpy2-{{ version }}.tar.gz
+  sha256: 159033cc0b81f9ac6f803ce50d39363cb704f64bb99d829630d13eaf42a30efc
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [python_impl == 'pypy']
+  skip: true  # [py<310 or win or osx]
+
+requirements:
+  host:
+    - python
```

Comment:
```suggestion
    - python >=3.10
```

Reply:
In the next revision (linux only), this change will not be needed.

Reply:
no need for version pin now